python/ray/util/accelerators/accelerators.py

```diff
@@ -8,6 +8,13 @@
 INTEL_MAX_1550 = "Intel-GPU-Max-1550"
 INTEL_MAX_1100 = "Intel-GPU-Max-1100"
 INTEL_GAUDI = "Intel-GAUDI"
+AMD_INSTINCT_MI100 = "AMD Instinct MI100"
+AMD_INSTINCT_MI250x = "AMD Instinct MI250X"
+AMD_INSTINCT_MI250 = "AMD Instinct MI250X / MI250"
+AMD_INSTINCT_MI210 = "AMD Instinct MI210"
+AMD_INSTINCT_MI300x = "AMD Instinct MI300X OAM"
+AMD_RADEON_R9_200_HD_7900 = "AMD Radeon R9 200 / HD 7900"
+AMD_RADEON_HD_7900 = "AMD Radeon HD 7900"
```

Comment:
As an convention, we don't have space for the accelerator type. Could you change space to `-`? e.g. `AMD-Radeon-R9-200-HD-7900`